Bio/NaiveBayes.py

```diff
@@ -51,23 +52,23 @@ class NaiveBayes(object):
     """
 
     def __init__(self):
+        """Initialize the class."""
         self.classes = []
         self.p_conditional = None
         self.p_prior = []
         self.dimensionality = None
 
 
 def calculate(nb, observation, scale=False):
-    """Calculate ``log P(class|observation)`` for each class.
+    """Calculate the logarithmic conditional probability for each class.
 
-        - nb          - A NaiveBayes classifier that has been trained.
-        - observation - A list representing the observed data.
-        - scale       - Boolean to indicate whether the probability should be
-          scaled by ``P(observation)``.  By default, no scaling is done.
+    - nb          - A NaiveBayes classifier that has been trained.
+    - observation - A list representing the observed data.
+    - scale       - Boolean to indicate whether the probability should be
+    scaled by ``P(observation)``.  By default, no scaling is done.
```

Comment:
This is probably now invalid indentation as RST, please check with e.g. http://rst.ninjs.org/